firestore/cloud-client/snippets.py

```diff
@@ -883,3 +883,65 @@ def update_document_increment(db):
 
     washington_ref.update("population", firestore.Increment(50))
     # [END fs_update_document_increment]
+
+
+# [START fs_counter_classes]
+import random
+from google.cloud import firestore
+
+
+class Shard(object):
+    """
+    Shard is a single counter, which is used in a group
```

Comment:
Looking at the other implementations, both this version and go have different style comments from the other languages. They also are a bit hard to understand.

`A shard is a distributed counter. Each shard can support being incremented once per second. Multiple shards are needed within a Counter to allow more frequent incrementing.`

Reply:
Well, comments in my [first commit in Go](https://github.com/GoogleCloudPlatform/golang-samples/pull/959/commits/18d2ee61634c4c33e41cab05677a1f65dd3866ff) was completely similar with comments from other samples. Review lead me to this. I don't want to say, that someone is wrong here, I'm trying to say, that I'm just a guest, doing what written rules and (after all) reviewers say. It'll be right to discuss comments with them, I think

Reply:
@crwilcox, pushed more detailed comments